include/roboteam_ai/stp/StpInfo.h

```diff
@@ -6,11 +6,11 @@
 #define RTT_STPINFO_H
 
 #include "world/Field.h"
-#include "world_new/views/BallView.hpp"
-#include "world_new/views/RobotView.hpp"
+#include "world/views/BallView.hpp"
+#include "world/views/RobotView.hpp"
 
 namespace rtt::ai::stp {
-
+namespace world = ::rtt::world;
```

Comment:
I don't have a huge problem with this but this kind of naming is confusing in my opinion. It looks like you're saying a = a + b. Not sure if there is a nicer way to do this?

Reply:
It was giving me compiler errors for `rtt::world` and somehow couldn't find it, so i'd have ot mark it as a global namespace `::rtt::world` which is ugly as hell to use everywhere so i just did this ^^